commands/class-make-block-command.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * Make Command class.
+ *
+ * @package Creode Blocks
+ */
+
+/**
+ * Command in WP_CLI to make a new block.
+ */
+class Make_Block_Command {
+	/**
+	 * Creates a new block class within your theme.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <name>
+	 * : The name of the block to create e.g. "Creode Footer".
+	 *
+	 * [--theme=<active-theme>]
+	 * : The slug of the theme to create the block in.
+	 *
+	 * ## EXAMPLES
+	 *
+	 * wp make-block "Creode Footer" --theme=creode
+	 *
+	 * @when after_wp_load
+	 *
+	 * @param array $args List of arguments.
+	 * @param array $optional_args List of optional arguments.
+	 */
+	public function __invoke( $args, $optional_args ) {
+		$theme_base_path = $this->get_theme_base_path( $optional_args );
+
+		$block_label = $args[0];
+
+		$block_class_name = $this->generate_block_class_name( $block_label );
+
+		$block_slug_name = $this->generate_block_slug_name_from_class( $block_class_name );
```

Comment:
Why does this function require a class name specifically? Shouldn't it require $block_label?